geoips/compare_outputs.py

```diff
@@ -214,95 +214,42 @@ def images_match(output_product, compare_product, fuzz="5%"):
     bool
         Return True if images match, False if they differ
     """
-    out_diffimg = get_out_diff_fname(compare_product, output_product)
     exact_out_diffimg = get_out_diff_fname(
         compare_product, output_product, flag="exact_"
     )
-
-    call_list = [
-        "compare",
-        "-verbose",
-        "-quiet",
-        "-metric",
-        "ae",
-        "-fuzz",
-        fuzz,
-        output_product,
-        compare_product,
-        out_diffimg,
-    ]
-
-    exact_call_list = [
-        "compare",
-        "-verbose",
-        "-quiet",
-        "-metric",
-        "ae",
-        output_product,
-        compare_product,
-        exact_out_diffimg,
-    ]
-
-    LOG.info("**Running %s", " ".join(call_list))
-    fullimg_ret = subprocess.run(
-        call_list, stdout=subprocess.PIPE, stderr=subprocess.STDOUT
-    )
-    LOG.debug(fullimg_ret.stdout)
-
+    from PIL import Image
+    import numpy as np
+    from pixelmatch.contrib.PIL import pixelmatch
+
+    LOG.info("**Comparing output_product vs. compare product")
+    out_img = Image.open(output_product)
+    comp_img = Image.open(compare_product)
+    diff_img = Image.new(mode="RGB", size=comp_img.size)
+    diff_arr = np.abs(np.array(comp_img) - np.array(out_img))
+    num_pix_mismatched = pixelmatch(out_img, comp_img, diff_img, includeAA=True,
+                                    alpha=0.33, threshold=0.05)
+    fullimg_retval = 0 if np.all(diff_arr == 0) and num_pix_mismatched == 0 else 1
```

Comment:
Just as a heads up, I can't figure out one problem when running tests/integration_tests/full_test.sh. I get a 1 on the check_code.sh, which outputs the 'warning' `would reformat /local/home/evrose54/geoips/geoips_packages/geoips/geoips/compare_outputs.py`. It doesn't say what line however. I thought it was because of this line, however I changed it to 
`if if np.all(diff_arr == 0) and num_pix_mismatched == 0: 
fullimg_retval = 0
else:
fullimg_retval = 1
` statement on my local branch, and still got the same output. Does anyone know why this output is occurring? Can't seem to figure this one out. All other tests pass on my end.

Reply:
Found out how to fix it. Resolved.